faker/providers/user_agent/__init__.py

```diff
@@ -13,20 +13,39 @@
 class Provider(BaseProvider):
     user_agents = (
         'chrome', 'firefox', 'internet_explorer', 'opera', 'safari',
+        'microsoft_edge',
     )
 
     windows_platform_tokens = (
         'Windows 95', 'Windows 98', 'Windows 98; Win 9x 4.90', 'Windows CE',
         'Windows NT 4.0', 'Windows NT 5.0', 'Windows NT 5.01',
         'Windows NT 5.1', 'Windows NT 5.2', 'Windows NT 6.0', 'Windows NT 6.1',
-        'Windows NT 6.2',
+        'Windows NT 6.2', 'Windows NT 6.3', 'Windows NT 10.0',
     )
 
     linux_processors = ('i686', 'x86_64',)
 
     mac_processors = ('Intel', 'PPC', 'U; Intel', 'U; PPC',)
 
-    langs = ('en-US', 'sl-SI', 'it-IT',)
+    # Language codes from ISO 639-1
+    langs = (
```

Comment:
Using just language might be technically better, but I think the purpose of `Faker` is to generate data as close as possible to what the user may actually encounter in practice.

In other words, it's technically correct and considered best practice to use just the language, but how _common_  is it in practice for user agent string? Does it happen often?

For the same reason, I agree that the `locale` method needs to be fixed.